src/test/java/org/alliancegenome/curation_api/base/BaseITCase.java

```diff
@@ -344,6 +344,25 @@ public GOTerm createGoTerm(String curie, String name, Boolean obsolete) {
 		return response.getEntity();
 	}
 
+	public GOTerm createGOTermWithSubsets(String curie, String name, List<String> subsets) throws Exception {
```

Comment:
Perhaps we could refactor the existing createGoTerm method to take subsets and ancester as input parameters and then just have methods that pass in nulls as appropriate for when we don't need those.

Reply:
Yeah, I can definitely do that.